ports/curl/portfile.cmake

```diff
@@ -37,20 +37,29 @@ set(BUILD_OPTIONS
     -DCURL_DISABLE_DOH=ON
     -DCURL_DISABLE_FILE=OFF
     -DCURL_DISABLE_FTP=ON
+    -DCURL_DISABLE_GETOPTIONS=ON
     -DCURL_DISABLE_GOPHER=ON
     -DCURL_DISABLE_HSTS=OFF
     -DCURL_DISABLE_HTTP=OFF
+    -DCURL_DISABLE_HTTP_AUTH=OFF
     -DCURL_DISABLE_IMAP=ON
     -DCURL_DISABLE_LDAP=ON
     -DCURL_DISABLE_LDAPS=ON
+    -DCURL_DISABLE_MIME=OFF
     -DCURL_DISABLE_MQTT=ON
+    -DCURL_DISABLE_NETRC=ON
+    -DCURL_DISABLE_NTLM=ON
+    -DCURL_DISABLE_PARSEDATE=OFF
     -DCURL_DISABLE_POP3=ON
     -DCURL_DISABLE_PROXY=OFF
     -DCURL_DISABLE_RTSP=ON
+    -DCURL_DISABLE_SHUFFLE_DNS=ON

```

Comment:
I don't see anything using CURLOPT_DNS_SHUFFLE_ADDRESSES so this seems safe to me.